api/tests/core/mails/transactional/pro/invoice_available_to_pro_test.py

```diff
@@ -17,13 +18,17 @@ def test_send_email(self):
 
         send_invoice_available_to_pro_email(invoice)
 
+        period_start, period_end = get_invoice_period(invoice.date)
+
         assert len(mails_testing.outbox) == 1
         assert (
             mails_testing.outbox[0].sent_data["template"] == TransactionalEmail.INVOICE_AVAILABLE_TO_PRO.value.__dict__
         )
         assert mails_testing.outbox[0].sent_data["To"] == "pro@example.com"
         assert mails_testing.outbox[0].sent_data["params"] == {
             "MONTANT_REMBOURSEMENT": 10,
+            "PERIODE_DEBUT": period_start.strftime("%d-%m-%Y"),
+            "PERIODE_FIN": period_end.strftime("%d-%m-%Y"),
```

Comment:
(détail) On reproduit dans ce test l'implémentation (et idem avec l'appel à `get_invoice_period()` plus haut). Peut-être qu'on pourrait forcer la date dans l'appel à `InvoiceFactory` et vérifier la période renvoyée avec des valeurs définies. C'est d'ailleurs ce qu'on fait déjà pour le montant.

```suggestion
            "PERIODE_DEBUT": period_start.strftime("15-01-2023"),
            "PERIODE_FIN": period_end.strftime("30-01-2023"),
```